test/api.mocha.coffee

```diff
@@ -259,6 +259,12 @@ describe "API", ->
               expect(body.err).to.be "Task not found."
               done()
 
+    describe "Anonymized User", ->
+      it "/api/v2/user/anonymized", (done) ->
+        request.get(baseURL + "/user/anonymized").set("Accept", "application/json").end (res) ->
+          expect(res.statusCode).to.be 200
```

Comment:
This test says that the route returned a 200, which is great, but it should also inspect the res.body and make sure that no personal details are included.